tools/system_libs.py

```diff
@@ -725,7 +725,7 @@ def get_cflags(self):
     if self.is_mt:
       cflags += ['-pthread', '-sWASM_WORKERS']
     if self.is_ww:
-      cflags += ['-sWASM_WORKERS']
+      cflags += ['-sSHARED_MEMORY=1']
```

Comment:
I think this is now the only remaining controversial change.   This does mean that no `-ww` libraries can ever use `__EMSCRIPTEN_WASM_WORKERS__` which seems odd.   So perhaps revert this line too.

The net result of this PR would then me "SHARED_MEMORY uses WASM_WORKERS libraries for now, but most them will work just fine even without the wasm workers library itself".  

I think we could land that and then iterate from there if it doesn't work in all cases.

Reply:
In the long run I agree we want to almost all libraries to be built only with `SHARED_MEMORY` and then only libwasm_workers and libpthreads  (and perhaps one or two more) to be build with specificly wasm workers or pthreads.

Reply:
Ah, I see. I'll look into changing those